mautic/mautic
# 4.4.4 to 5.x

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

This PR is merging changes from https://github.com/mautic/mautic/releases/tag/4.4.4 into 5.x

This is the last time we (or rather I?) "merge" an M4 release into the 5.x branch. When I attempted the merge, there were 191 conflicting files. So I rather added all the remotes, fetched and cherry-picked all the commits from the 4.4.4 release as it was less time consuming.

Since now we must be very strict on what PRs we merge into the 4.4 branch. It **must** be merged into 5.x branch first. Not only opened, but merged. Otherwise PRs like https://github.com/mautic/mautic/pull/11556 will happen which is useless and requires more time from maintainers to fix that.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Test the PRs from https://github.com/mautic/mautic/releases/tag/4.4.4



Gitpod link: [https://gitpod.io/#https://github.com/mautic/mautic/pull/11655
](https://gitpod.io/#https://github.com/mautic/mautic/pull/11655)
